### DISCUSSION TITLE
`postCreateCommand` logs stalling/crashing midway through build

### DISCUSSION THREAD

**Discussion by scottrobertson** [community/community#5279] [Codespaces] [answered]
When creating a codepace and using `postCreateCommand` that has a lot of output, it will randomly stop outputting and just stall until the `postCreateCommand` is finished. You can then view the rest of the logs in the Creation Log. 

For example, here you can see that the terminal has just stalled, but the creation log shows it all.

![Screenshot 2021-08-25 at 01 12 21](https://user-images.githubusercontent.com/68361/130706209-c12c0719-549d-4c71-b94a-4761292fcbef.png)

I am not convinced this is linked to line count, as it's quite inconsistent. It may be linked to how fast the logs are generated.

Here is a repo to replicate the issue: https://github.com/scottrobertson/codespaces-log-issue

**Comment by jedieaston** [comment]
I'm also getting this trying to setup a Rails project with a ton of dependencies, although it stops randomly. It builds and runs totally fine with VSCode's local Dev Containers extension.

(It's made it super hard to determine if it's safe to bring up the project because I can't tell if the postCreateCommand has finished. It'd be nice if there was an option to stay on the "Building your codespace..." screen until the postCreateCommand has completed, but that's besides the issue).

**Reply by scottrobertson** [reply]
Agreed. Blocking the whole UI until that is finished would be ideal (but with log output).

**Comment by Chuxel** [comment, marked as answer]
This appears to be bug - we'll work on fixing it!

**Reply by jkeech** [reply]
@scottrobertson, this bug should be fixed now! Please let us know if you continue to see this issue. Thanks!

**Reply by metaskills** [reply]
I can confirm I've seen this issue and can also confirm it is fixed. Thanks y'all!!
